### ISSUE TITLE
Forms - CP - General View Error

### ISSUE THREAD

**Issue by The-Dan-Main** [verbb/formie#2023]
### Describe the bug

After updating to the newest version 3.0.2, in the control-panel, no forms can be changed or viewed. There is a javascript error, which is perhaps preventing the view.

### Steps to reproduce

1. Install Craft & Formie
2. Create a new form
3. enter the form

Here is the error logged in the console:

Uncaught TypeError: Cannot read properties of null (reading 'nextSibling')
    at nextSibling (runtime-dom.esm-bundler.js:40:31)
    at Et (runtime-core.esm-bundler.js:6600:14)
    at Ct (runtime-core.esm-bundler.js:6571:9)
    at Ve (runtime-core.esm-bundler.js:6549:9)
    at zt (runtime-core.esm-bundler.js:6617:15)
    at Ve (runtime-core.esm-bundler.js:6518:24)
    at m (runtime-core.esm-bundler.js:5286:15)
    at be (runtime-core.esm-bundler.js:6656:7)
    at mount (runtime-core.esm-bundler.js:3921:20)
    at t.mount (runtime-dom.esm-bundler.js:1511:19)

### Form settings

- Multi-page form: does not matter, both is issued
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

### Craft CMS version

5.3.4

### Plugin version

3.0.2

### Multi-site?

does not matter, both is issued

**Comment by denisyilmaz** [comment]
Same here, error started after updating to version `2.1.25` for craft 4

**Comment by denisyilmaz** [comment]
It seems to be related to `nystudio107/craft-plugin-vite` which got updated alongside `verbb/formie` and `craftcms/cms` as a peerDependency. 

I downgraded `verbb/formie` (2.1.24) and `craftcms/cms` (4.11.2) to its previous versions and explicitly set the `nystudio107/craft-plugin-vite` dependency to the previous version with  `ddev composer require nystudio107/craft-plugin-vite=4.0.11`. This fixed the issue in the CP. So other plugins might be affected by this update (formie/craftcms) as well.
